### ISSUE TITLE
[Shaman] Anuniaq bugged and not giving me daily

### ISSUE THREAD

**Issue by zstango** [dalaranwow/dalaran-wow#5724]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I'm a level 79 shaman who finished all of the Kalu'ak quests, yet the only daily I cannot get is the one from Anuniaq. I did the Borean tundra series first, then dragonblight, and finished with the Howling Fjord. 
**Expected behaviour**: Tell us what should happen instead
I should unlock all dailys
I have missed out on like 2,000 reputation at least from not being able to do this daily. 
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
The quest is available. Please check WotLK sources to verify that you have completed all the prerequisites of the daily.
Closing.

**Comment by Chenocide** [comment]
Some of the Kalu'ak quests gets unlocked at higher reputation tiers. I don't know if this is blizzlike but that's how I remember it when I farmed the reputation.
